### ISSUE TITLE
Septic emboli

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6539]
**Preferred term label:** Septic emboli

**Synonyms**

**Definition (free text, please give PubMed ID)**

Embolisation of intravascular thrombus containing microorganisms into the distant tissues via arterial system PMID: 31751017

**Parent term (use hpo.jax.org/app)**
Thromboembolism HP:0001907

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
see https://pubmed.ncbi.nlm.nih.gov/31751017/

**Comment by eswietlik** [comment]
I have added PMID

**Comment by pnrobinson** [comment]
adding term
